editoast/src/views/single_simulation.rs

```diff
@@ -35,11 +35,20 @@ pub enum SingleSimulationError {
     WrongCoreResponseFormat,
 }
 
-pub fn routes() -> impl HttpServiceFactory {
-    scope("/single_simulation").service((standalone_simulation,))
+crate::routes! {
+    "/single_simulation" => {
+        standalone_simulation,
+    }
+}
+
+schemas! {
+    ScheduleArgs,
```

Comment:
You don't need to add ScheduleArgs to components/schemas since you inlined it in SingleSimulationRequest. Also duplicated :D